Modules/Quotas/src/Quotas.Infrastructure/Persistence/Repository/RelationshipsRepository.cs

```diff
@@ -19,7 +19,8 @@ public async Task<uint> Count(string createdBy, DateTime createdAtFrom, DateTime
         var relationshipsCount = await _relationshipsReadonly
```

Comment:
I think we should unit test the whole method. It's becomes too complicated.

Reply:
You should add a test for the behavior I described in https://github.com/nmshd/backbone/pull/597#discussion_r1584243259